numba/npyufunc/dufunc.py

```diff
@@ -0,0 +1,33 @@
+from __future__ import absolute_import, print_function, division
+import numpy
+
+from numba import jit, typeof, utils
+from .. import numpy_support
```

Comment:
Mixing import styles isn't very pretty. All the modules above come from the "numba" toplevel, right?